volttron/platform/packaging.py

```diff
@@ -688,6 +723,10 @@ def main(argv=sys.argv):
 
         if opts.subparser_name == 'package':
             whl_path = create_package(opts.agent_directory, wheelhouse=opts.dest, identity=opts.vip_identity)
+        elif opts.subparser_name == 'install-agent':
+            install_args = ['python', 'scripts/install-agent.py']
+            install_args.extend(sys.argv[2:])
+            subprocess.Popen(install_args)
```

Comment:
Do a wait() so that you wait for the process to complete. Without this, when I run the vpkg install-agent command the command never exits. It hangs.